static/js/ducks/source.js

```diff
@@ -140,10 +140,10 @@ messageHandler.add((actionType, payload, dispatch, getState) => {
   const { source } = getState();
 
   if (actionType === REFRESH_SOURCE) {
-    const loaded_obj_id = source ? source.id : null;
+    const loaded_obj_key = source ? source.internal_key : null;
```

Comment:
simplify with `?.`

Reply:
Sure, this just follows what was done before, but I can update